### PR TITLE
#13 Adds env2 for env variables

### PR DESCRIPTION
**Pull request by Cleop**
To be merged after #15 

#13 Adds env2 for env variables

### CODE REVIEW

File: bin/www [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+const env = require("env2")(".env");
```
Comment (SimonLab) [inline bin/www]: I don't think the `env` variable is used after in the file?
I think the path to the `.env` file is not correct. As the working directory is `bin/www` will need to go to the parent directory (root of the project) to find the file. Howerver by default I think `env2` will try to find a .env file in the project, see https://github.com/dwyl/env2/blob/df6a56900b711b44cb0fc6e083ead272c21ce47d/lib/env.js#L25 so the path to the file name is not necessary in this case, so we could just use `require("env2"))()`

I think I would also require the `env2` module in `app.js` which is the main file of the application, in a similar way the article has done with `dotenv`

Comment (Cleop) [reply bin/www]: It's used to set the port @SimonLab: `var port = normalizePort(process.env.PORT || '3000');`, no?

Originally I had it as `('../.env')` but I got a warning error from env2 saying the file could not be found. This way I get no error. 

But I think your suggestion for putting it in `app.js` is a good idea so I'll trial that.

Comment (Cleop) [reply bin/www]: It only works with the `'.env'` file path in `app.js` I'll recommit. It still gives the warning without the file path.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require("env2")(".env");
```
Comment (nelsonic) [inline app.js]: 👍

**Pull request merged**